.github/workflows/build-base-images.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build base images for SAVE execution
+
+on:
+  schedule:
+    - cron: '0 4 * * 1'
+  workflow_dispatch:
+
+jobs:
+  build_base_images:
+    name: Build base images and push to ghcr.io
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # This should replicate list of SDKs from com.saveourtool.save.domain.Sdk
+        include:
+          - base_image_name: opendjk
+            base_image_tag: 8
+          - base_image_name: opendjk
+            base_image_tag: 9
+          - base_image_name: opendjk
+            base_image_tag: 10
+          - base_image_name: opendjk
+            base_image_tag: 11
+          - base_image_name: opendjk
+            base_image_tag: 12
+          - base_image_name: opendjk
+            base_image_tag: 13
+          - base_image_name: opendjk
+            base_image_tag: 14
```

Comment:
Only half of those images can be used directly, others probably have different tag format. Anyway, we may want to support only several LTS versions
https://github.com/saveourtool/save-cloud/pkgs/container/save-base/versions
https://github.com/saveourtool/save-cloud/actions/runs/2839233453